### PR TITLE
docs: landing integrations section

### PR DESCRIPTION
**Pull request by ozkalai**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

Creating an `integrations` section in the new landing.

### Test plan (required)

This development does not affect test suites.

### Self Check before Merge

Please check all items below before review.

-   [X] Corresponding issues are created/updated or not needed
-   [X] Docs are updated/provided or not needed
-   [X] Examples are updated/provided or not needed
-   [X] TypeScript definitions are updated/provided or not needed
-   [X] Tests are updated/provided or not needed
-   [X] Changesets are provided or not needed

### CODE REVIEW

File: documentation/src/local-json/inegrations.json [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                {
+                    "name": "tspvivek",
+                    "url": "https://github.com/tspvivek"
+                }
+            ]
+        }
+    ],
+    "frameworks": [
+        {
+            "name": "NEXTJS",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:139]:
```suggestion
            "name": "Next.js",
```

File: documentation/src/local-json/inegrations.json [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            "icon": "/integrations/nextjs.svg",
+            "description": "The React Framework for Production",
+            "url": "https://nextjs.org",
+            "status": "stable"
+        },
+        {
+            "name": "REMIX",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:146]:
```suggestion
            "name": "Remix",
```

File: documentation/src/local-json/inegrations.json [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+            "icon": "/integrations/remix.svg",
+            "description": "Remix is a full stack web framework that lets you focus on the user interface and work back through web standards to deliver a fast, slick, and resilient user experience.",
+            "url": "https://remix.run",
+            "status": "stable"
+        },
+        {
+            "name": "GATSBY",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:153]:
```suggestion
            "name": "Gatsby",
```

File: documentation/src/local-json/inegrations.json [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        {
+            "name": "Supabase Data Provider",
+            "icon": "/integrations/supabase.svg",
+            "description": "Supabase is an open source Firebase alternative. Start your project with a Postgres Database, Authentication, instant APIs, Realtime subscriptions and Storage.",
+            "url": "https://supabase.com/",
+            "status": "stable"
+        },
+        {
+            "name": "Hasura GraphQL Data Provider",
+            "icon": "/integrations/hasura.svg",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:63]: The detail in the middle of the Hasura logo is missing

File: documentation/src/local-json/inegrations.json [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+            "icon": "/integrations/kbar.svg",
+            "description": "kbar is a fully extensible command+k interface for your site.",
+            "url": "https://kbar.vercel.app",
+            "status": "stable"
+        }
+    ],
+    "live-providers": [
+        {
+            "name": "Ably Live Provider",
+            "icon": "/integrations/ably.svg",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:186]: the top of the logo is cut off on the page

File: documentation/src/local-json/inegrations.json [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+            "name": "Strapi Data Provider",
+            "icon": "/integrations/strapi.svg",
+            "description": "Strapi is the leading open-source headless CMS. It’s 100% JavaScript, fully customizable and developer-first.",
+            "url": "https://strapi.io",
+            "status": "stable"
+        },
+        {
+            "name": "Supabase Data Provider",
+            "icon": "/integrations/supabase.svg",
+            "description": "Supabase is an open source Firebase alternative. Start your project with a Postgres Database, Authentication, instant APIs, Realtime subscriptions and Storage.",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:57]: "realtime" must be bold

File: documentation/src/local-json/inegrations.json [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+            "name": "Nhost Data Provider",
+            "icon": "/integrations/nhost.svg",
+            "description": "The Open Source Firebase Alternative with GraphQL",
+            "url": "https://nhost.io/",
+            "status": "stable"
+        },
+        {
+            "name": "Appwrite Data Provider",
+            "icon": "/integrations/appwrite.svg",
+            "description": "Appwrite is a self-hosted backend-as-a-service platform that provides developers with all the core APIs required to build any application.",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:78]: It should be mentioned in the statement that we have realtime support

File: documentation/src/local-json/inegrations.json [hunk 8]
```diff
@@ -0,0 +1,233 @@
+{
+    "ui-framework-packages": [
+        {
+            "name": "ANT DESIGN",
+            "icon": "/integrations/antd.svg",
+            "description": "a fully featured Ant Design System UI Framework support. Includes +20 Table, Form, Select, Menu, Layout, Notification and CRUD components and hooks.",
+            "url": "https://ant.design",
```
Comment (omeraplak) [inline documentation/src/local-json/inegrations.json:7]: I think our npmjs packages should be given as the link

File: documentation/src/local-json/inegrations.json (hunk 5 above)
Comment (ozkalai) [reply documentation/src/local-json/inegrations.json:186]: The original design has the cut on the top and also the edges of logo

File: documentation/src/components/integrations/card/index.tsx [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+                    {status === "soon" ? (
+                        <div>
+                            <ComingSoon />
+                        </div>
+                    ) : (
+                        <>
+                            <p className={styles.cardDescription}>
+                                {description}
+                            </p>
+                        </>
```
Comment (biskuvit) [inline documentation/src/components/integrations/card/index.tsx:59]:
```suggestion
                            <p className={styles.cardDescription}>
                                {description}
                            </p>
```

**Pull request merged**